crates/ark/src/data_explorer/format.rs

```diff
@@ -673,6 +674,26 @@ mod tests {
         })
     }
 
+    #[test]
+    fn test_s3_formatting() {
+        r_task(|| {
+            if !package_is_installed("survival") {
+                return;
+            }
+
+            let data =
```

Comment:
I am not sure if we can write a test for the Variables pane that is as directly helpful as this one, but it would be great if so!

Reply:
We could similar tests in: 

https://github.com/posit-dev/ark/blob/1bed8ebeaa0010f727413766efe54510774dd1ad/crates/harp/src/vector/formatted_vector.rs#L273-L284